bin/setup_configuration.sh

```diff
@@ -9,4 +9,5 @@ set -e
 ${SCRIPTPATH}/wait_for_db.sh
 
 src/manage.py migrate
+${SCRIPTPATH}/load_fixtures.sh
```

Comment:
I added this because otherwise it won't be possible to run the setup configuration part for authorizations, because the catalogi do not exist yet